tests/chainer_tests/link_hooks_tests/test_spectral_normalization.py

```diff
@@ -35,6 +41,8 @@ def test_invalid_shaped_weight(self):
 
 class BaseTest(object):
 
+    allclose_options = {'atol': 1e-6, 'rtol': 1e-4}
```

Comment:
Could you avoid setting a mutable object to an attribute?  How about setting in `__init__` or `setUp`.  (The subclasses might have to call `super`'s method.)

Reply:
I forgot to remove this.
I set this because I mistakenly used `numpy.testing.assert_allclose` instead of `testing.assert_allclose` because I overlooked it.